### ISSUE TITLE
Ebuild and first release

### ISSUE THREAD

**Issue by TheChymera** [IBT-FMI/gebuilder#30]
As the package approaches some general-purpose usability, we should be looking to provide it via a more popular overlay, or directly via the main Gentoo overlay.

I have set-up a [live ebuild on my overlay](https://github.com/TheChymera/overlay/blob/master/app-misc/gebuilder/gebuilder-9999.ebuild), and as soon as this works reliably we can release a ~stable version. Perhaps this would also be a good time for implementing a rudimentary test suite? e.g. at least for stemgentoo or for creating images of a gebuilder-single-purpose system.

@Doeme would you be interested in testing the ebuild and trying to write some basic tests?

**Comment by Doeme** [comment]
Sure, why ever not. Let's try to get this package up to speed

**Comment by Doeme** [comment]
What I don't like about the ebuild: non-privileged users can execute the `gebuild`script and get some cryptic error messages smacked in their face.

How can we prevent this?
* Moving `gebuild` into `/usr/sbin`, which is only in root's path. Contra: AFAIK programs like this don't usually belong into `/usr/sbin`
* chmod'ing `gebuild` to 0700. Script is then still in non-root users $PATH, but at least executing it yields `permission denied`
* Checking `$USER` for `!= root` at the beginning of `gebuild`, and bail out with a more graceful error message

Otherwise, the ebuild installed flawlessly and the stemgentoo is already set up

**Comment by TheChymera** [comment]
Is there any reason why we wouldn't allow users to create gebuild systems?

**Comment by Doeme** [comment]
Yes, chroot'ing is a privileged operation and therefore can not be performed by any other than root.

Maybe, at some point, we could consider instead of chrooting starting a lxc container (which would be a plus due to better isolation, see also #32). But I haven't looked into that, so I can't say whether this is feasible.

**Comment by TheChymera** [comment]
Too bad :( well, ok, we can add this to the 2.0 milestones, and in the meantime, since we know explicitly that the user has to be **root** regardless of permissions, I think

> Checking $USER for != root at the beginning of gebuild, and bail out with a more graceful error message 

makes the most sense.
